app/src/main/java/org/oppia/app/administratorcontrols/administratorcontrolsitemviewmodel/AdministratorControlsAccountActionsViewModel.kt

```diff
@@ -0,0 +1,4 @@
+package org.oppia.app.administratorcontrols.administratorcontrolsitemviewmodel
+
+/**  [ViewModel] for the recycler view in [AdministratorControlsFragment]. */
```

Comment:
Extra space at start. 
`/** [ViewModel] for the recycler view in [AdministratorControlsFragment]. */`

Reply:
Fixed in #740